packs/tests/actions/chains/test_quickstart_rules.yaml

```diff
@@ -28,7 +28,23 @@ chain:
         params:
             env:
               ST2_AUTH_TOKEN: "{{token}}"
-            cmd: "st2 rule create /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml &> /dev/null"
+            cmd: "st2 rule create /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml -j | grep name"
+        on-success: sleep
+    -
+        name: sleep
+        ref: core.local
+        params:
+            env:
+              ST2_AUTH_TOKEN: "{{token}}"
+            cmd: "sleep 5"
+        on-success: test_webhook_list    
+    -
+        name: test_webhook_list
+        ref: core.local
+        params:
+            env:
+              ST2_AUTH_TOKEN: "{{token}}"
+            cmd: "st2 webhook list -j | grep sample"
```

Comment:
It seems like that json query (http://stedolan.github.io/jq/) would come handy in the tests and make them more robust and fail safe.